Replace "could" and "might", Clauses 16-32. (#4386)
cplusplus/draft: source/threads.tex

\pnum
\begin{note}

> If the clock is not synchronized with a steady clock, e.g., a CPU time clock,
> it is possible that these timeouts do not provide useful functionality.

Reviewer: This feels a little clearer and less awkward:

```suggestion
If the clock is not synchronized with a steady clock, e.g., a CPU time clock, these
timeouts can fail to provide useful functionality.
```

Something like "...these timeouts are not guaranteed to provide..." would be even better, but I'm not sure if that passes ISO muster.

Author: Thanks, the suggestion is a nice improvement. About "not guaranteed to provide useful functionality", I suspect that that additional strengthening wouldn't help much if we don't also define "useful functionality", so I think what we have now will do the job.

\begin{note}
These apply regardless of the provided \tcode{policy} argument, and
even if the corresponding \tcode{future} object is moved to another thread.

> It is possible for \tcode{f} not to be called at all,

Reviewer: ```suggestion
However, it is possible for \tcode{f} not to be called at all,
```
